docs/developer/develop-a-dashboard-plugin.md

```diff
@@ -0,0 +1,112 @@
+---
+sidebar_label: Developing a plugin
+title: Developing a Dashboard plugin
+description: Learn how to develop a plugin for the DHIS2 Dashboard app.
+id: develop-a-dashboard-plugin
+---
+
+Developing a dashboard plugin is almost the same as developing a regular DHIS2 custom app. However, there are some differences in how you structure your plugin and how you configure it to work with the DHIS2 Dashboard app. These are outlined in the few steps below.
+
+Here is an example of a dashboard plugin developed according to this documentation.
+
+![Climate data dashboard plugin](./resources/dashboard-plugin-example.png)
+
+## Step 1: Create a new DHIS2 custom app
+
+To create a new DHIS2 custom app, you can use the D2 CLI.
+
+The D2 CLI helps you with many things, such as setting up a new project, building your app, and deploying it to your DHIS2 instance.
+
+Please read our documentation on the [D2 CLI](/docs/quickstart/quickstart-web) to learn more about how to use it.
+
+```sh
+d2 app scripts init my-dashboard-plugin
+```
+
+## Step 2: Add a plugin entrypoint
+
+To make your custom app a form field plugin, you need to add a plugin entrypoint inside the `d2.config.js` file.
```

Comment:
I'm not familiar with a "form field" plugin. What is that?

Reply:
Sorry, copy paste leftover.